src/logic/operate.js

```diff
@@ -0,0 +1,29 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  const x = Big(numberOne);
+  const y = Big(numberTwo);
+  let r = 0;
+  switch (operation) {
+    case '+':
+      r = Number(x) + Number(y);
+      break;
+    case '-':
+      r = x - y;
+      break;
+    case 'x':
+      r = x * y;
+      break;
+    case '÷':
+      r = x / y;
+      break;
+    case '%':
+      r = x / 100;
+      break;
+    default:
+      r = 'fail';
```

Comment:
Job well done but you don't have to do the calculating yourself, that where the use of `big` comes in -> [big js library](https://github.com/favio41/big-js#readme)
example: case division will be -> `total.minus(next)`